### ISSUE TITLE
2D verion of S3DIS

### ISSUE THREAD

**Issue by Rabbia-Hassan** [CVMI-Lab/PLA#28]
Hello !
Your work is very interesting. Thank you for sharing it.
I reuest you to please provide the link for the 2d version of S3DIS dataset. I checked the Dataset.md file but the link is very confusing. Please provide the link for the 2d version of the dataset.

Regards
Rabbia Hassan

**Comment by Dingry** [comment]
I think the provided link is quite clear. We cannot directly spread the dataset link due to license restrictions. 
![image](https://github.com/CVMI-Lab/PLA/assets/39176662/7fdc7b14-f0fb-4d70-ba62-77bcb3f28bce)

**Comment by Areeba-Raza** [comment]
thank you for the response. Th confusion is that this dataset is 766Gb and the other is 110Gb as mentioned. Can you specify please that which one to download ?

**Comment by Dingry** [comment]
I use the 'without global_xyz' version. I will clarify this in README later. Thanks for your advice.

**Comment by Rabbia-Hassan** [comment]
Thank you very much for the response. Also please tell me, did you use the complete 1.3 TB sized version of scanner_v2 dataset to generate the captions ? Because its not given also and it is slightly confusing. Any clarification from your side will be greatly helpful. Thank you for your time.

**Comment by Dingry** [comment]
We use only 25k images.
![image](https://github.com/CVMI-Lab/PLA/assets/39176662/26aa9a0a-55db-4415-a2ba-73a7ba2daefb)

**Comment by Rabbia-Hassan** [comment]
right, many thanks, also I checked the cpationed data version of S3DIS that you provided in your drive is not full S3DIS dataset. It doesnt have the captions for Area 5 point clouds. That will be great if you can clarify . Many thanks again.

**Comment by Dingry** [comment]
Area 5 is the validation set of S3DIS. We only need captioning on the training set. Please refer to our paper for details.
